### DISCUSSION TITLE
Looking for ESH on Mapbox Fees

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1347] [Q&A] [answered]
We are looking at using 12 Step Meeting List as a solution to manage the meeting list for our Intergroup.

We are a little concerned about the MapBox fees as it seems difficult to assess how much we might end up paying monthly if anything.

We currently have 16 meetings, 4 of which are currently in-person. We are currently getting about 60 visitors to our site each day.

Based on rough math it seems like we would not hit the 50,000 map views allowed on the free account but we wanted to get some experience, strength and hope on what kinds of fees larger intergroups are incurring on a monthly basis, just to be safe.

Thank you so much
Will M

**Comment by joshreisner** [comment]
hi will, we've never heard of any site hitting its mapbox quota and being charged. san francisco / marin intergroup (aasfmarin.org/meetings) is also using mapbox and has not hit the limit yet

that said, we don't have a good way of knowing how # of site visitors translates into # of api requests. i'd guess roughly 1.5 loads per site visitor? that would be ~2,700 api loads, which is well under the threshold. but curious to know how that works out for you

**Reply by willinrecovery** [reply]
This is helpful. Are you one of the developers?

**Reply by joshreisner** [reply]
of the plugin, yes

**Reply by willinrecovery** [reply]
Thank you Josh. San Francisco / Marin definitely has more meetings than our Intergroup. 

Do you know how long they have been using the plugin?

**Reply by joshreisner** [reply]
i think they started using mapbox and TSML UI on their site around 2018

**Reply by willinrecovery** [reply, marked as answer]
Thank you again, Josh. I am guessing that if they were going to get a charge, it would have happened by now.
